raiden-dapp/src/App.vue

```diff
@@ -57,10 +57,10 @@ export default class App extends Vue {
     );
   }
 
-  async connect() {
-    this.connecting = true;
+  async connect(subkey?: true) {
+    this.connecting = true && !subkey;
```

Comment:
Nitpick: It still needs to Connect to Metamask (if it's not) to prompt signing the subkey message, so I'd say you could still have the loading indicator even with subkey on.

Reply:
Ok, but I have to add a new loading indicator since that one displays a spinner inside the button even when the subkey text link is pressed. But no problem, I'll look into adding a second spinner.